processor/src/main/java/org/mapstruct/ap/internal/model/Mapper.java

```diff
@@ -129,16 +135,34 @@ public Builder extraImports(SortedSet<Type> extraImportedTypes) {
             return this;
         }
 
+        public Builder implName(String implName) {
+            this.implName = "default".equals( implName ) ? "*Impl" : implName;
```

Comment:
Could you extract constants for "*" (or whatever we end up using for instead)? Makes it easier to change :)